ArduCopter/mode_auto.cpp

```diff
@@ -614,17 +614,17 @@ void PayloadPlace::start_descent()
     auto *wp_nav = copter.wp_nav;
 
     // set horizontal speed and acceleration limits
-    pos_control->set_max_speed_accel_NE_cm(wp_nav->get_default_speed_xy(), wp_nav->get_wp_acceleration());
-    pos_control->set_correction_speed_accel_NE_cm(wp_nav->get_default_speed_xy(), wp_nav->get_wp_acceleration());
+    pos_control->set_max_speed_accel_NE_cm(wp_nav->get_default_speed_NE_cms(), wp_nav->get_wp_acceleration_cmss());
+    pos_control->set_correction_speed_accel_NE_cm(wp_nav->get_default_speed_NE_cms(), wp_nav->get_wp_acceleration_cmss());
 
     // initialise the vertical position controller
     if (!pos_control->is_active_NE()) {
         pos_control->init_NE_controller();
     }
 
     // set vertical speed and acceleration limits
-    pos_control->set_max_speed_accel_U_cm(wp_nav->get_default_speed_down(), wp_nav->get_default_speed_up(), wp_nav->get_accel_z());
-    pos_control->set_correction_speed_accel_U_cmss(wp_nav->get_default_speed_down(), wp_nav->get_default_speed_up(), wp_nav->get_accel_z());
+    pos_control->set_max_speed_accel_U_cm(wp_nav->get_default_speed_down_cms(), wp_nav->get_default_speed_up_cms(), wp_nav->get_accel_U_cmss());
```

Comment:
Using both `up` and `U` is a little confusing.

Reply:
I don't think it is too bad here. Up and Down are explicitly the maximum speeds in those directions where "U" is the axis.

As you say, in the longer term I would like to have a set of defaults at the top of each libary file that says the defaults are NED and m, radians. All variables using alternatives will be labled _deg for example.

I am leaning towards radians being the default internal unit for angle and degrees being the default parameter unit.

Reply:
My impression is that People were happy with this on the dev call.